### PR TITLE
feat(2092): [2] Dropdown build parameters

### PR DESCRIPTION
**Pull request by yoshwata**
## Context

Add feature of https://github.com/screwdriver-cd/screwdriver/issues/2092

## Objective

**Please merge after https://github.com/screwdriver-cd/ui/pull/575**
Add about dropdown build parameters.

## License

~~https://github.com/screwdriver-cd/ui/pull/573~~
https://github.com/screwdriver-cd/ui/pull/575

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: docs/user-guide/configuration/parameters.md [hunk 1]
```diff
@@ -9,14 +9,15 @@ toc:
       active: true
 ---
 ## Defining Parameters
-There are 2 ways to define parameters, you can see them both in the example below.
+There are 3 ways to define parameters, you can see them both in the example below.
```
Comment (tkyi) [inline docs/user-guide/configuration/parameters.md]:
```suggestion
There are 3 ways to define parameters, you can see them in the example below:
```

File: docs/user-guide/configuration/parameters.md [hunk 2]
```diff
@@ -9,14 +9,15 @@ toc:
...
 
 ```yaml
 parameters:
     nameA: "value1"
     nameB:
         value: "value2"
         description: "description of nameB"
+    nameC: ["value1", "value2"]
```
Comment (tkyi) [inline docs/user-guide/configuration/parameters.md:20]: should we add that the default will be the first value in the array?

Comment (yoshwata) [reply docs/user-guide/configuration/parameters.md:20]: Yes, and It is important thing. I add about that.

**Review by wahapo — APPROVED** [review]

**Review by jithine — APPROVED** [review]

File: docs/user-guide/configuration/parameters.md (hunk 2 above)
Comment (yoshwata) [reply docs/user-guide/configuration/parameters.md:20]: Hi @tkyi. Is it OK?

File: docs/user-guide/configuration/parameters.md [hunk 3]
```diff
@@ -37,6 +38,13 @@ parameters:
 
 These two examples above are equivalent.
 
+You can also define parameters with array to pick up from some candidates. The top value of the array is picked up as default, e.g `value1` is selected as default in the example blow:
```
Comment (tkyi) [inline docs/user-guide/configuration/parameters.md]:
```suggestion
You can also define parameters in an array for drop down options. The first value in the array is picked up as default, e.g `value1` is selected as default in the example below:
```

File: docs/user-guide/configuration/parameters.md (hunk 2 above)
Comment (tkyi) [reply docs/user-guide/configuration/parameters.md:20]: small comment!

File: docs/user-guide/configuration/parameters.md (hunk 3 above)
Comment (yoshwata) [reply docs/user-guide/configuration/parameters.md]: Thank you. I fixed it.

**Review by kumada626 — APPROVED** [review]

**Review by tkyi — APPROVED** [review]

**Pull request merged**